packages/cli/src/util/progress.ts

```diff
@@ -16,6 +16,7 @@ export function showProgress({
   let current = 0;
   let last = 0;
   let lastProcessTime: number = Date.now();
+  let internalId: NodeJS.Timeout;
```

Comment:
could use a more descriptive name for the interval, I think it will also not be just an ID but the whole `Timeout` object

Reply:
It's common convention among JS community to suffix output of `setTimeout` or `setInterval` as `Ids` because these are just the pointers to v8 internal timers.

Reply:
I changed the name to be more descriptive.

Reply:
> It's common convention among JS community to suffix output of setTimeout or setInterval as Ids because these are just the pointers to v8 internal timers

gotcha, thanks for pointing that out, I didn't know about that